Gemfile

```diff
@@ -59,9 +59,9 @@ gem 'feedback', '~> 0.3.1'
 gem 'mortgage_calculator', '~> 1.3.5'
 gem 'payday_loans_intervention', '~> 1.4'
 gem 'pensions_calculator', '~> 1.0.0'
+gem 'quiz', git: 'git@github.com:moneyadviceservice/quiz.git'
```

Comment:
@benbarnett there's another gem in RubyGems so to avoid name clash we used this. We could just as well have defined the source